sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -605,7 +602,7 @@ def str(self):
     return _DeferredStringMethods(self._expr)
 
 
-for name in ['apply', 'map', 'transform']:
+for name in ['apply', 'map']:
```

Comment:
Should we just expand this out to 

```
apply = frame_base._elementwise_method('apply')
map = frame_base._elementwise_method('map')
```

in the body of DeferredSeries?

Reply:
Done. Also added a jira/TODO for transform